chain/pool/txpool.go

```diff
@@ -209,7 +213,13 @@ func (tp *TxnPool) GetTransaction(hash common.Uint256) *transaction.Transaction
 		}
 		return true
 	})
-	return found
+	if found != nil {
+		return found
+	}
+	if np, ok := tp.NanoPayTxs.Load(hash); ok {
```

Comment:
I think we should search NanoPayTxs before TxLists because the former one is much faster while TxLists needs to be traversed.

Reply:
done